user/salt-lts/files/salt-api

```diff
@@ -0,0 +1,6 @@
+type = process
```

Comment:
please use correct style for all of these services (i.e. proper usage of `=` vs `:` and so on)